src/foundry/foundry.js/applications/formApplication.d.ts

```diff
@@ -45,7 +46,7 @@ declare abstract class FormApplication<
    * The values of this Array are inner-objects with references to the MCE editor and other metadata
    * @defaultValue `{}`
    */
-  editors: Partial<Record<string, FormApplication.FormApplicationEditor>>;
+  editors: Record<string, FormApplication.FormApplicationEditor>;
```

Comment:
Why remove the `Partial`?

Reply:
Because this succeeds:
`expectType<Record<string, unknown>>((null as unknown) as Partial<Record<string, unknown>>);`

Reply:
yeah, for unknown, it doesn't matter. Actually, for all super types of `undefined`, it doesn't matter. For all others, there a very relevant difference. In the case we are talking about here, doing
```ts
const anEditor: FormApplication.FormAcpplicationEditor = someFormApplication.editors.someEditorThatForSureDoesntExist;
```
works perfectly fine. If you use `Partial`, the type would need to be `FormApplication.FormAcpplicationEditor | undefined`, which is the correct thing imo—we don't know which keys exist and which don't.